src/app/_helpers/filter-conf.ts

```diff
@@ -0,0 +1,68 @@
+import { FilterExecution } from '../_models/filterExecution';
+
+export const filterConf: FilterExecution[] = [
+  {
+    label: 'workflow',
+    name: 'WORKFLOW',
+    multi: true,
+    options: [
+      { value: 'enrich' },
+      { value: 'import a' },
+      { value: 'import b' },
+      { value: 'normalise' },
+      { value: 'preview' },
+      { value: 'validate' },
+      { value: 'transform' },
+    ],
+  },
+  {
+    label: 'date',
+    name: 'DATE',
+    options: [
+      { value: 'last 24' },
+      { value: 'last week' },
+      { value: 'last month' },
+      {
+        value: 'from',
+        input: {
+          id: 'date-from',
+          type: 'date',
+          fnOnSet: (val: string): void => {
+            const dateTo = document.querySelector('#date-to');
+            if (dateTo) {
+              val ? dateTo.setAttribute('min', val) : dateTo.removeAttribute('min');
+            }
+          },
+        },
+        group: 'date-pair',
+      },
+      {
+        value: 'to',
+        input: {
+          id: 'date-to',
```

Comment:
I can select a date-to that is before the date-from.

Reply:
It's possible to circumvent the max / min values using the keyboard only - not by clicking on the calendar.  This is the soft enforcement of the html 5 input - the default native behaviour.  Hard enforcement would require the disabling key events, which I don't think is preferable.